bignum/src/commonMain/kotlin/com/ionspin/kotlin/bignum/integer/BigInteger.kt

```diff
@@ -611,8 +611,14 @@ class BigInteger internal constructor(wordArray: WordArray, requestedSign: Sign)
         return BigInteger(arithmetic.and(this.magnitude, other.magnitude), sign)
     }
 
+    /** Returns a new BigInt with bits combining [this], [other] doing a bitwise `|`/`or` operation. Forces sign to positive. */
```

Comment:
Why are we forcing the sign to positive here?

Reply:
I think I understand the reasoning behind forcing to positive. I added a TODO and I'll reopen the issue 290 after this is merged.